build-system/tasks/extension-helpers.js

```diff
@@ -630,6 +630,18 @@ async function buildBentoCss(name, versions, minifiedAmpCss) {
 async function buildNpmBinaries(extDir, name, options) {
   let {npm} = options;
   if (npm === true) {
+    // remap all shared modules to the @bentoproject/core package and declare as external
+    // todo(kvchari): currently only used for bento standalone, investigate using for preact & react
+    const bentoEntryPoint = await getBentoBuildFilename(
```

Comment:
super nit: can this be renamed `entryPoint`?

Reply:
I named specifically `bentoEntryPoint` to be unambiguous since the `preact` and `react` binaries have different entry points